src/main/java/com/amihaiemil/zold/RtWallet.java

```diff
@@ -84,11 +84,26 @@ public String getId() throws IOException {
     }
 
     @Override
-    public double balance() {
-        throw new UnsupportedOperationException(
-            "Not yet implemented. If you can contribute please, do it at "
-            + "https://github.com/amihaiemil/zold-java-client"
+    public double balance() throws IOException{
```

Comment:
@ammaratef45 , could you please make a unit test for this method (you may have seen already that the coverage check is not happy)?

Reply:
@denis-konovalyenko 
I guess adding a todo for unit tests is okay (because this class has no unit testing at all and this should be an issue)

Reply:
@ammaratef45 , I have replied on this in the scope of another conversation.

Reply:
@denis-konovalyenko yes, separate todo is fine :)